libs/app-shell/src/lib/sitemap.js

```diff
@@ -97,10 +99,14 @@ const sitemap = [
       {
         label: 'Alliance User Community',
         route: 'https://groups.google.com/a/alliancegenome.org/g/community',
+        external: true,
+
```

Comment:
@adamgibs these are not served in the app these are generated in the sitemap.xml file no need to add this here.